src/tags/control/Ranker.js

```diff
@@ -31,20 +31,24 @@ const RankerItemModel = types
   }));
 
 /**
- * Ranker tag, used for ranking models.
+ * Use the Ranker tag to rank the results from models. This tag uses the "prediction" field from a labeling task instead of the "data" field to display content for labeling on the interface. Carefully structure your labeling tasks to work with this tag. See [import pre-annotated data](../guide/predictions.html).
```

Comment:
since line breaks in the comments aren't well-handled in the md changeover, this is now the longest line ever, i'm sorry